[Linux] - Build error with gcc-15.0.1

## Bug Description

Build error with gcc-15.0.1 on Fedora Rawhide

## System Information

- **Operating System**: Fedora rawhide x86_64
- **BambooTracker Version**: 0.6.5
- **Build Type**: locally

The error message during the build:
```
g++ -c -fPIC -include utility -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1  -m64 -march=x86-64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -mtls-dialect=gnu2 -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer  -I. -Iinstrument -Imodule -I../submodules/emu2149/src -I../submodules/RtAudio/src -I../submodules/RtMidi/src -I/usr/include/qt5 -I/usr/include/qt5/QtWidgets -I/usr/include/qt5/QtGui -I/usr/include/qt5/QtCore -I. -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o swap_instruments_command.o command/instrument/swap_instruments_command.cpp
make[1]: Leaving directory '/builddir/build/BUILD/BambooTracker-0.6.5-build/BambooTracker/BambooTracker'
In file included from instrument/instrument.hpp:36,
                 from instrument/instruments_manager.hpp:33,
                 from command/instrument/swap_instruments_command.hpp:10,
                 from command/instrument/swap_instruments_command.cpp:6:
instrument/instrument_property_defs.hpp:51:6: error: expected identifier before â€˜:â€™ token
   51 | enum : uint8_t
      |      ^
instrument/instrument_property_defs.hpp:51:6: error: expected unqualified-id before â€˜:â€™ token
instrument/instrument.hpp:225:21: error: â€˜uint8_tâ€™ was not declared in this scope
  225 |         std::vector<uint8_t> getRawSample() const;
      |                     ^~~~~~~
instrument/instrument.hpp:39:1: note: â€˜uint8_tâ€™ is defined in header â€˜<cstdint>â€™; this is probably fixable by adding â€˜#include <cstdint>â€™
   38 | #include "bamboo_tracker_defs.hpp"
  +++ |+#include <cstdint>
   39 | 
instrument/instrument.hpp:225:28: error: template argument 1 is invalid
  225 |         std::vector<uint8_t> getRawSample() const;
      |                            ^
instrument/instrument.hpp:225:28: error: template argument 2 is invalid
instrument/instrument.hpp:297:21: error: â€˜uint8_tâ€™ was not declared in this scope
  297 |         std::vector<uint8_t> getRawSample(int key) const;
      |                     ^~~~~~~
instrument/instrument.hpp:297:21: note: â€˜uint8_tâ€™ is defined in header â€˜<cstdint>â€™; this is probably fixable by adding â€˜#include <cstdint>â€™
instrument/instrument.hpp:297:28: error: template argument 1 is invalid
  297 |         std::vector<uint8_t> getRawSample(int key) const;
      |                            ^
instrument/instrument.hpp:297:28: error: template argument 2 is invalid
make[1]: *** [Makefile:2489: swap_instruments_command.o] Error 1
```

--- a/BambooTracker/instrument/instrument_property_defs.hpp
+++ b/BambooTracker/instrument/instrument_property_defs.hpp
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#include <cstdint>
+
 #include "sequence_property.hpp"
 
 using FMOperatorSequenceUnit = InstrumentSequenceBaseUnit;

